sticht/slack.py

```diff
@@ -35,7 +35,11 @@
     construct_conn_msg = None
 
 SLACK_WEBHOOK_STREAM = 'stream_slack_incoming_webhook'
-SCRIBE_ENV = 'uswest1-prod'
+SCRIBE_ENV = {
+    'region': 'uswest1-prod',
```

Comment:
I think this should be uswest2-prod. I believe the latest version of scribereader doesn't even support uswest1-prod anymore.